src/html/blocks/menu/menu.html

```diff
@@ -0,0 +1,35 @@
+<nav class="menu page__menu menu_@@mods">
+    <div class="list action-list">
+        <a href="#" class="list-item list-item_small menu__settings">
+            Settings
+        </a>
+
+        <a href="/task-list.html" class="list-item menu__today">
+            today
+            <div class="badge-group">
+                <i class="badge badge_circle color_yellow"></i>
+                <i class="badge badge_circle color_red"></i>
+                <i class="badge badge_circle color_green"></i>
+            </div>
+        </a>
+
+        <a href="/week-task-list.html" class="list-item menu__next-seven-days">
+            next 7 days
+        </a>
+    </div>
+
+    <div class="separator"> projects</div>
```

Comment:
This is header or title of project list, so will be better to change class name.